_shared/project/Makefile

```diff
@@ -105,15 +104,18 @@ frontend-test:
 {% set target = 'test-py%s'|format(python_version|pyformat(PyFormats.MAJOR_MINOR_FMT)) %}
 .PHONY: {{ target }}
 $(call help,make {{ target }},"run the unit tests in Python {{ python_version|pyformat(PyFormats.MAJOR_DOT_MINOR_FMT) }}")
-coverage: {{ target }}
 {{ target }}: python
 	@pyenv exec tox -qe py{{ python_version|pyformat(PyFormats.MAJOR_MINOR_FMT) }}-tests
 {%- endfor %}
 
 .PHONY: coverage
 $(call help,make coverage,"run the tests and print the coverage report")
 coverage: python
-	@pyenv exec tox -qe coverage
+{%- if cookiecutter._directory == 'pyramid-app' %}
+	@pyenv exec tox -qe 'tests,coverage'
+{%- else %}
+	@pyenv exec tox --parallel -qe 'tests,py{{ "{" }}{{ python_versions()|rest|pyformat(PyFormats.MAJOR_MINOR_FMT)|separator(",") }}{{ "}" }}-tests,coverage'
+{%- endif %}
```

Comment:
tox is aware of the dependency of `coverage` on `{pyXY-}tests`. If you just run `tox -e coverage` it'll just run the coverage report. But if you run `tox -e coverage,tests,py39-tests,py38-tests,py37-tests` it'll reorder the environments for you to run `coverage` last. If you add `--parallel` to that then it knows that all the tests can be run in parallel but `coverage` can't start until the tests finish.